doc/source/getting-started/installing/linux.rst

```diff
@@ -0,0 +1,87 @@
+Install PyWorkbench locally in Linux
+####################################
+
+This guideline covers the local installation of PyWorkbench in Linux platforms.
+
+.. note::
+
+    This guide assumes that you have a local installation of STK and a valid
+    license in your machine.
```

Comment:
```suggestion
    The following instructions assume that you have a local installation of STK and a valid
    license in your machine.
```
Valid license of what? STK or Ansys Workbench. I assume it is the second. Either way, specify what license is needed.